app/controllers/admin_controller.rb

```diff
@@ -17,6 +17,22 @@ def holdings
 
   private
 
+  def mediated_pages
+    if params[:expired]
+      expired_mediated_pages
+    else
+      active_mediated_pages
+    end
```

Comment:
If you're renaming things, I wonder if:
 -  the `.where(origin: params[:id]).order(:origin).page(page).per(per)` below should be moved into `mediated_pages`,
 - `where(origin: params[:id])` is a scope, and if
 - `order(:origin)` is redundant/unnecessary?